src/SwipeableDrawer/SwipeableDrawer.js

```diff
@@ -355,6 +361,11 @@ SwipeableDrawer.propTypes = {
    * to promote accidental discovery of the swipe gesture.
    */
   disableDiscovery: PropTypes.bool,
+  /**
+   * If `true`, swipe to open is disabled. This is useful in browsers where swiping triggers
+   * navigation actions. Swipe to open disabled on iOS browsers by default.
```

Comment:
Uhm… Yes, we're indeed missing a verb here. Should I move the description somewhere else so that the table doesn't explode? Maybe we should fix the table instead? :wink:

Reply:
That said, I think that the table could wait and that this fix should be included in the next release.

Reply:
My comment on the API table layout wasn't meant for this pull-request. It's a broader issue. We should hold this PR back for it.